web-ui/app.js

```
@@ -86,10 +86,12 @@ document.addEventListener('DOMContentLoaded', () => {
                 configTemplateDiffHasChangesValue: false,
                 configTemplateDiffFingerprint: '',
                 _configTemplateDiffPreviewRequestToken: null,
+                configTemplateDiffConfirmEnabled: true,
                 codexApplying: false,
                 _pendingCodexApplyOptions: null,
                 agentsContent: '',
                 agentsPath: '',
```

Biome (2.4.11)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)